tests/spec/w3c/headers-spec.js

```diff
@@ -276,6 +276,49 @@ describe("W3C — Headers", function() {
       expect(secondEditor.localName).toEqual("dd");
       expect(secondEditor.textContent).toEqual("NAME2");
     });
+
+    it("adds support for configs in different languages", async () => {
+      const ops = makeStandardOps();
+      const newProps = {
+        specStatus: "REC",
+        editors: [
+          {
+            name:
+              "<span lang='ja'>阿南 康宏</span> (Yasuhiro Anan), (<span lang='ja'>第１版</span> 1st edition)",
+            company: "Microsoft",
+          },
+        ],
+        formerEditors: [
+          {
+            name:
+              "<span lang='ja'>阿南 康宏</span> (Yasuhiro Anan), (<span lang='ja'>第１版</span> 1st edition)",
+            company: "Microsoft",
+          },
+        ],
+      };
+      Object.assign(ops.config, newProps);
+      const doc = await makeRSDoc(ops);
+      var spanName = document.createElement("span");
```

Comment:
Close, but here you need to reach into `doc` and find the `dd`.

Reply:
See how to do it here: https://github.com/OrionStar25/respec/blob/5a9be94c046738954ac6867ad124753a25b153e4/tests/spec/w3c/headers-spec.js#L185